Wrappers/Python/cil/optimisation/functions/KullbackLeibler.py

```diff
@@ -455,7 +455,8 @@ def gradient(self, x, out = None):
 
         if self.mask is not None:
             kl_gradient_mask(x.as_array(), self.b.as_array(), out_np, self.eta.as_array(), self.mask)         
-        kl_gradient(x.as_array(), self.b.as_array(), out_np, self.eta.as_array())            
+        else:
+            kl_gradient(x.as_array(), self.b.as_array(), out_np, self.eta.as_array())            
```

Comment:
As far as I can see this is the only change. We should have a unittest for this.

Reply:
done :)